packages/api/src/utils/addUser.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from 'express';
+import { User } from '@hangar/database';
+
+export const addUser = async ( req: Request, res: Response, next:Function ) => {
+  const email = req.session?.email;
+  const userQuery = { email };
+  
+  if (!email) {
+    // User does not have a valid session
+    res.sendStatus(401);
+    return;
+  }
```

Comment:
Can you utilize `id` instead? Same exact query, just leveraging the id instead of email